GasGrid/OtherTools/Mapping and Location Tools/mapbox_CMCL/index.html

```diff
@@ -190,9 +190,6 @@
                 var name = e.features[0].properties["Offtake Point (License Name)"];
                 var type = e.features[0].properties["Type of Offtake"];
 				
-				// Update the side panel
```

Comment:
No need for side panel for offtakes due to lack of real-time information, information such as type of offtake probably best suited for a label